scripts/spelling_gate_test_support.py

```
@@ -0,0 +1,109 @@
+"""Controlled-process fixtures for the tracked-file spelling gate."""
+
+from __future__ import annotations
+
+import json
+import os
+from pathlib import Path
+import shutil
+import subprocess
+import sys
+import textwrap
+from typing import Mapping
```

Ruff (0.16.3)

[warning] Import from `collections.abc` instead: `Mapping`

Import from `collections.abc`

(UP035)